lib/shop_invader/services/elastic_service.rb

```diff
@@ -142,6 +142,8 @@ def build_params(conditions)
               end
             elsif %w(gt gte lt lte).include?(op)
               params[:bool][:filter] << { range: { name => { op => value } } }
+            elsif %w(in).include?(op)
```

Comment:
not a big deal but %w is to make list of strings. For a single string IMHO it is cleaner to have
`elsif op == "in"`

Reply:
@rvalyi I'm not a ruby developer :smirk: I'll change the code

Reply:
@rvalyi done